src/properties.rs

```diff
@@ -1,21 +1,21 @@
 use std::io::BufReader;
 
+use crate::winstream::WinStream;
```

Comment:
I kinda tried to put all internal imports after external imports although there was an exception. Meaning, I'd be happier to keep WinStream after windows imports.

Reply:
Sorry it's the cargo fmt command that moved everything.